client/electron/electron-builder.json

```diff
@@ -37,11 +37,15 @@
     "maintainer": "Jigsaw LLC",
     "target": [{
       "arch": [
-        "x64"
+        "x64",
+        "arm64"
```

Comment:
As mentioned earlier, maybe we can drop the AppImage here, it's more complicated.

Reply:
done